hedera-node/configuration/mainnet/application.properties

```diff
@@ -1 +1,12 @@
-
+autoRenew.targetTypes=
+contracts.knownBlockHash=055fac8eb788872957dfee45b7d40f7c94d66ea0a75ec703b993d57e441de574@33476932
+contracts.localCall.estRetBytes=4096
+hedera.allowances.isEnabled=true
+scheduling.whitelist=CryptoTransfer,ConsensusSubmitMessage,TokenBurn,TokenMint,CryptoApproveAllowance
+sigs.expandFromLastSignedState=false
+staking.fees.stakingRewardPercentage=10
+staking.maxDailyStakeRewardThPerH=6_849
+staking.maxStakeRewarded=650000000000000000
+staking.rewardBalanceThreshold=8500000000000000
+staking.rewardRate=48630136986000
```

Comment:
same for this property `staking.rewardRate`. It has been deleted

Reply:
If this is no longer applicable to the `0.41.x` release then we should remove and seek approval from @netopyr

Reply:
Removed.